testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt

```diff
@@ -107,7 +107,7 @@ class ProfileSettingsActivity : BaseActivity() {
                     inputStream?.use {
                         val bitmap = BitmapFactory.decodeStream(inputStream)
                         val byteArrayOutputStream = ByteArrayOutputStream()
-                        bitmap.compress(Bitmap.CompressFormat.JPEG, 50, byteArrayOutputStream)
+                        bitmap.compress(Bitmap.CompressFormat.PNG, 100, byteArrayOutputStream)
```

Comment:
what happened to JPEG? Just curious.

Reply:
Yeah, we can keep either `JPEG` / `PNG`, so, I have just wanted to be the same with [SettingsFragment.kt#L350](https://github.com/rakutentech/android-miniapp/blob/master/testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapptabs/fragments/SettingsFragment.kt#L350).